rz-sbc/rzsbc_yocto.sh

```diff
@@ -100,6 +103,17 @@ check_pkg_require(){
     [ ${check} -ne 0 ] && echo "Package check failed. Fix erros and copy dependencies here." && exit
 }
 
+check_patch_require(){
+    echo "Checking patch dependencies..."
+    for patch in "${patch_list[@]}"; do
```

Comment:
It’s fine to create a separate file for new patches. However, since other components like library packages or BSP meta-layers also change frequently, it is better to create a tracking file and use some handler functions to manage them.

Reply:
Also, is json a good option? I think yaml is better.

Reply:
Okay, then I'd close this PR.